src/tests/Microsoft.PowerFx.Interpreter.Tests/DateTimeTests.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class DateTimeTests
+    {
+        private static readonly bool IsLocalUTC = TimeZoneInfo.Local.BaseUtcOffset == TimeSpan.Zero;
+
+        [Fact]
```

Comment:
Can we also test `DateTimeValue()`, `Today()`, and `IsToday()` functions?

- the function `DateTimeValue` would be internally evaluated to c# DateTime object of the correct time zone. Like `DateTimeValue("2022-10-01")` should be evaluated as 2022-10-01 in UTC when `Symbol.SetTimeZone(TimeZoneInfo.Utc)`
- `IsToday()` and `Today()` should honor the setting by SetTimeZone()

Reply:
Added the new tests. Feel free to point out any specifics on those tests!